assets/models/system/salesforce-xgen-7b-4k-base/spec.yaml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: salesforce-xgen-7b-4k-base
+path: ./
+properties:
+  SHA: 7dde7ac26ddcb679e0429fed1b6f751c1ffab1fb
```

Comment:
![image](https://github.com/Azure/azureml-assets/assets/61145377/9356cbb7-39d9-44b4-bc3b-a0a134c55b27)

Reply:
What is expected here

Reply:
commit id was not correct. But saw it got updated. Thanks